appcues/src/main/java/com/appcues/ui/AppcuesActivity.kt

```diff
@@ -172,6 +173,7 @@ internal class AppcuesActivity : AppCompatActivity() {
                                                 // if we have contentPadding to apply from the WrapContent trait then we apply here
                                                 .then(if (contentPadding != null) Modifier.padding(contentPadding) else Modifier)
                                                 .padding(paddingValues = stepDecoratingPadding.paddingValues.value)
+                                                .testTag("page_$index")
```

Comment:
This is for some upcoming UI test work, so that we can accurately identify elements on a given page inside of a Pager control - came from some good tips I learned here https://akjaw.com/wiriting-ui-tests-for-pager-layouts-jetpack-compose/